packages/polymath-ui/src/components/STOStatus/STOStatus.js

```diff
@@ -133,23 +137,47 @@ export default class STOStatus extends Component<Props> {
               </div>
             </div>
           </div>
-          {countdownProps != null && (
+          {isSaleComplete ? (
             <div className="pui-countdown-container">
-              <Countdown
-                deadline={countdownProps.deadline}
-                title={countdownProps.title}
-                buttonTitle={
-                  notPausable
-                    ? undefined
-                    : this.props.isStoPaused
-                    ? 'RESUME STO'
-                    : 'PAUSE STO'
-                }
-                handleButtonClick={this.props.toggleStoPause}
-                isPaused={this.props.isStoPaused}
-                pausable={!notPausable}
-              />
+              <ContentBox
+                style={{
+                  borderRadius: '10px',
+                  borderTop: '15px solid #00AA5E',
+                  width: '350px',
+                }}
+              >
+                <p align="center">
```

Comment:
Same here, use new component. `align` HTML property is invalid HTML.
`<Paragraph textAlign="center">`